ledger-core/virtual/execute/execute_semiintegration_test.go

```diff
@@ -71,9 +72,12 @@ func TestSMExecute_Semi_IncrementPendingCounters(t *testing.T) {
 		var (
 			authService authentication.Service = authentication.NewServiceMock(t)
 			catalog     object.Catalog         = catalogWrapper.Mock()
+
+			limiter = conveyor.NewParallelProcessingLimiter(0)
```

Comment:
to avoid messing it up with notation of other semaphore types where limit is explicit, I'd suggest to create 2 separate constructors please:
NewParallelProcessingLimiter(<limit>)
NewParallelProcessingLimiterByCPU()

Reply:
done